pkg/mod/mod.go

```diff
@@ -20,7 +20,7 @@ import (
 
 // Spaces are never allowed in usernames,
 // so this string will never be a valid username.
-var CensoredUsername = "Unknown Woogler"
+var CensoredUsername = "Unwoogler"
```

Comment:
- comment mismatch now
https://github.com/domino14/liwords/blob/9988ce136a9f488397fc29256d8800fbe21cffd4/pkg/mod/mod.go#L21-L23
- need to patch this
https://github.com/domino14/liwords/blob/e2b70b0cef09d3ea8b92dec2ca12e71f1a4f7b68/pkg/registration/register.go#L27-L30
- Unwoogler now shows up in rematch streak, are we now changing all "Unknown Woogler"?
- full name, country, (avatar?) still show up; turns out the `GAME_HISTORY_REFRESHER` in socket still has the real `userId`
- in game rooms, avatar letter now shows as '?' on player card, but as 'U' on moves list (seems PlayerAvatar is getting called with undefineds)

Reply:
It was decided that all instances of "Unknown Woogler" can be changed to "Unwoogler" to avoid whitespace problems. So it looks like there is still the GAME_HISTORY_REFRESHER socket events that need to be sanitized. Where is the call to PlayerAvatar happening?